tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -4707,31 +4707,31 @@ public function dataArrayFunctions(): array
 				'array_values($generalStringKeys)',
 			],
 			[
-				'non-empty-array<int|(literal-string&non-empty-string), stdClass>',
+				'array{foo: stdClass, 1: stdClass}',
```

Comment:
changed existing tests, to more specific types - as the PR gets better results in this cases.